python/TestHarness/TestHarness.py

```diff
@@ -562,6 +564,102 @@ def cleanup(self):
             # Perform any write-to-disc operations
             self.writeResults()
 
+    def findRacePartners(self):
```

Comment:
Shouldn't all three of these methods be offloaded to your RaceConditionChecker class too? The TestHarness should ideally just be querying the Scheduler, passing everything to the RaceConditionChecker and getting back results.

Reply:
Yeah, there should already be exact copies in the RaceConditionChecker class.  I thought I had removed this code from here, but I think I accidentally reverted the commit or undid it before committing or something like that.  I'll remove it remove this code and make sure it still works.